lib/ansible/modules/cloud/amazon/aws_kms.py

```diff
@@ -197,12 +203,12 @@ def do_grant(kms, keyarn, role_arn, granttypes, mode='grant', dry_run=True, clea
     try:
         if len(changes_needed) and not dry_run:
             policy_json_string = json.dumps(policy)
-            kms.put_key_policy(KeyId=keyarn, PolicyName='default', Policy=policy_json_string)
-    except:
-        raise Exception("{}: // {}".format("e", policy_json_string))
+    except Exception as e:
+            raise Exception("{}: // {}".format(e, policy_json_string))
```

Comment:
There is one problem with doing it this way.  If json.dumps() raises an exception then policy_json_string won't be set to a value.  Instead, you return the repr of the policy dict:
``` python
raise Exception("{0} // {1}".format(e, repr(policy))
```

(I also remembered that the "{}" format string isn't legal on python-2.6 so we have to use the numbers inside of them.